scripts/monorepo.sh

```diff
@@ -12,7 +12,8 @@ if [ ! "$BRANCH" = "master" ]; then
   HAS_CHANGES=$(git diff `git merge-base HEAD origin/master` --name-only -- $SHARED_CONFIG_FILES $SERVICE)
   if [ ! -n "$HAS_CHANGES" ]; then
     echo "No change in $SERVICE, skipping job."
-    circleci step halt
+  else
+    echo "changed"
```

Comment:
Can we change the circle config to just call `circleci step halt` if result is `changed` ?

Reply:
yep I added to circleci conf directly